blottertrax/database.py

```diff
@@ -11,10 +11,11 @@ class Database:
     sql = None
 
     def __init__(self):
-        self.sql = sqlite3.connect('{}/../database/submissions.db'.format(os.path.dirname(os.path.realpath(__file__))))
+        self.sql = sqlite3.connect('{}/../database/database.db'.format(os.path.dirname(os.path.realpath(__file__))))
```

Comment:
I agree on this but don't forget to copy the current `submissions.db` to `database.cb` before merging or every mod action will be performed twice for last submissions

Reply:
Ya I almost wrote something to check whether the user account had commented on/performed a mod action the post but decided it would be easier to just copy the DB over instead.

Reply:
I have some progress, splitting the submissions from the modmail. Don't have access to my old laptop with my testing configuration but I will do some testing this Sunday/Monday